tools/dataform_assertions/definitions/personal_info_assertion_unit_test.sqlx

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config {
+    type: "view",
+    schema: "dev_data_zone",
+    name: "personal_info_assertions_unit_test",
+    tags:["personal_info_assertions_unit_test","all_test"],
+    assertions: {
+        nonNull: ["test_name"],
+        rowConditions: 
+        [
+            personal_info_assertions.test_same_character_more_than_n_times("test_name",3),
+            personal_info_assertions.test_name_validity("test_name")
+        ]
+    }
+}
+
+/* This test should fail 2 assertions*/
+
+/*
+    test_name_validity
+    Return false if name is irregular
+*/
+SELECT "Alan" AS test_name UNION ALL    --This should pass 
+SELECT "Bob" AS test_name UNION ALL    --This should pass
+SELECT "yuy&^%" AS test_name UNION ALL             --This should fail
```

Comment:
Please rename test_name to 'input' and add a second column 'expected_output' with the static value TRUE or FALSE depending on the expected test result. You can then remove all comments that say --This should ...